### PR TITLE
First working test for codim=1

### PR DESCRIPTION
**Pull request by asartori86**

### CODE REVIEW

**Comment by asartori86** [comment]
very latest version of deal.II is required

File: include/interfaces/poisson_problem.h [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                                                           LinearOperator<LATrilinos::VectorType> &system_op,
+                                                           LinearOperator<LATrilinos::VectorType> &prec_op) const
+{
+
+  preconditioner.reset  (new TrilinosWrappers::PreconditionJacobi());
+  preconditioner->initialize(matrices[1]->block(0,0));
+  auto P = linear_operator<LATrilinos::VectorType::BlockType>(matrices[0]->block(0,0));
+
+  auto A  = linear_operator<LATrilinos::VectorType::BlockType>( matrices[0]->block(0,0) );
+
```
Comment (luca-heltai) [inline include/interfaces/poisson_problem.h:113]: In this example, can we avoid using a solver around the preconditioner? We already have a preconditioner... just use the preconditioner, not a solver around it...

```
auto P_inv = linear_operator<LATrilinos::VectorType::BlockType>(*preconditioner, matrices[0]->block(0,0));
```

or

```
auto P_inv = linear_operator<LATrilinos::VectorType::BlockType>(matrices[0]->block(0,0), *preconditioner);
```

I never remember the order of the exemplar VS the object...

File: source/pidomus.cc [hunk 2]
```diff
@@ -531,15 +533,16 @@ void piDoMUS<dim, spacedim, LAC>::refine_mesh ()
...
-                                          0,
-                                          0,
-                                          triangulation->locally_owned_subdomain());
+      KellyErrorEstimator<dim,spacedim>::estimate (interface.get_mapping(),
+                                                   *dof_handler,
+                                                   QGauss <dim-1> (fe->degree + 1),
+                                                   typename FunctionMap<spacedim>::type(),
+                                                   distributed_solution,
+                                                   estimated_error_per_cell,
+                                                   ComponentMask(),
```
Comment (luca-heltai) [inline source/pidomus.cc:542]: Can you open an issue for this component mask? The user should be able to select a component mask for the refinement...

**Comment by asartori86** [comment]
@luca-heltai così va bene?

**Comment by asartori86** [comment]
ho aperto issue #93

**Pull request merged**